[2025-02-22T21:00:01.492+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-22T21:00:01.517+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: london_weather_pipeline.extract_weather_data scheduled__2025-02-22T20:00:00+00:00 [queued]>
[2025-02-22T21:00:01.527+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: london_weather_pipeline.extract_weather_data scheduled__2025-02-22T20:00:00+00:00 [queued]>
[2025-02-22T21:00:01.527+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-22T21:00:01.543+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): extract_weather_data> on 2025-02-22 20:00:00+00:00
[2025-02-22T21:00:01.549+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=6386) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-22T21:00:01.550+0000] {standard_task_runner.py:72} INFO - Started process 6387 to run task
[2025-02-22T21:00:01.551+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'london_weather_pipeline', 'extract_weather_data', 'scheduled__2025-02-22T20:00:00+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/london_weather_pipeline/london_weather_etl.py', '--cfg-path', '/tmp/tmpjydj4ee2']
[2025-02-22T21:00:01.552+0000] {standard_task_runner.py:105} INFO - Job 51: Subtask extract_weather_data
[2025-02-22T21:00:01.616+0000] {task_command.py:467} INFO - Running <TaskInstance: london_weather_pipeline.extract_weather_data scheduled__2025-02-22T20:00:00+00:00 [running]> on host ee2081ae3a04
[2025-02-22T21:00:01.733+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='london_weather_pipeline' AIRFLOW_CTX_TASK_ID='extract_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-22T20:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-22T20:00:00+00:00'
[2025-02-22T21:00:01.734+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-22T21:00:01.753+0000] {base.py:84} INFO - Retrieving connection 'open_meteo_id'
[2025-02-22T21:00:02.317+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': 51.5, 'longitude': -0.120000124, 'generationtime_ms': 0.05042552947998047, 'utc_offset_seconds': 0, 'timezone': 'GMT', 'timezone_abbreviation': 'GMT', 'elevation': 16.0, 'current_weather_units': {'time': 'iso8601', 'interval': 'seconds', 'temperature': '°C', 'windspeed': 'km/h', 'winddirection': '°', 'is_day': '', 'weathercode': 'wmo code'}, 'current_weather': {'time': '2025-02-22T21:00', 'interval': 900, 'temperature': 9.8, 'windspeed': 8.1, 'winddirection': 225, 'is_day': 0, 'weathercode': 0}}
[2025-02-22T21:00:02.379+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-22T21:00:02.380+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=london_weather_pipeline, task_id=extract_weather_data, run_id=scheduled__2025-02-22T20:00:00+00:00, execution_date=20250222T200000, start_date=20250222T210001, end_date=20250222T210002
[2025-02-22T21:00:02.489+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-22T21:00:02.561+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-22T21:00:02.565+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
